packages/devtools-launchpad/bin/chrome-driver.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+
+const spawn = require('child_process').spawn;
+
+const chromeBinary = "/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome"
```

Comment:
nope - we should fix this separately. In fact, if you could fix it as I think you have linux that would be awesome